addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2644,6 +2644,9 @@ we-select.o_grid we-toggler {
     }
 
     &.o_we_grid_preview {
+        @include media-breakpoint-down(lg) {
+            height: 0;
+        }
```

Comment:
The explanation in the commit message is nice, but I think it would also be worth adding a comment around these lines explaining why we want to hide it, and why we use height instead of display, what do you think?